src/rgw/driver/rados/rgw_bucket.cc

```diff
@@ -2597,15 +2605,37 @@ int RGWMetadataHandlerPut_BucketInstance::put_checked(const DoutPrefixProvider *
 
   RGWSI_Bucket_BI_Ctx ctx(op->ctx());
 
-  return bihandler->svc.bucket->store_bucket_instance_info(ctx,
-                                                         entry,
-                                                         info,
-                                                         orig_info,
-                                                         false,
-                                                         mtime,
-                                                         pattrs,
-							 y,
-                                                         dpp);
+  //retry loop
```

Comment:
sorry, i think i was mistaken about the need for a retry loop within `RGWMetadataHandlerPut_BucketInstance`

that should be happening at a higher level - for example, metadata sync already loops over calls to `put()` in [RGWMetaSyncSingleEntryCR](https://github.com/ceph/ceph/blob/785c108/src/rgw/driver/rados/rgw_sync.cc#L1314-L1317)

it doesn't make sense to loop here in `put_checked()`, because we're just rewriting the same thing we read without the other modifications from `put_check()` above. the retries from RGWMetaSyncSingleEntryCR should do the right thing already

Reply:
done. I should have checked the sync call there. thanks!